scholia/app/templates/base.html

```diff
@@ -104,7 +106,7 @@
 
      });
 
-     var endpointUrl = 'https://query.wikidata.org/sparql';
+     var endpointUrl = '{{ webserviceURL }}sparql';
```

Comment:
This seems to be wrong.

Reply:
Fixed. 
I tested everything before committing. Everything seemed to work.
These URLs are spread all around. It's easy to miss. Thanks